### ISSUE TITLE
Apacer Panther DIMMs are not detected as DDR5

### ISSUE THREAD

**Issue by DB-47** [sebhildebrandt/systeminformation#845]
**Describe the bug**
My Apacer Panther RGB DDR5 DIMMs are detected as unknown type, when I attempt to list them using si.memLayout() function. 

**To Reproduce**
1. Use function si.memLayout(); , no furhter params and actions are needed.

**Current Output**
![image](https://github.com/sebhildebrandt/systeminformation/assets/34443035/1541af63-d66f-4086-86d9-bc2e1c3c283c)

**Expected behavior**
Those Apacer DIMM modules should be detected as proper DDR5

**Environment (please complete the following information):**
 - systeminformation package version:
 - OS: Windows 11 Pro N
 - DIMM used: Apacer Panther RGB DDR5 (listed as gd2.47312h.004)

**Additional context**
Official website of those DIMMs. https://www.apacer.com/en/product/personal-product/detail/personal_dram/panther_rgb_ddr5. Results from HWinfo app. 
![image](https://github.com/sebhildebrandt/systeminformation/assets/34443035/a1aa9e9a-0d90-41c5-b381-afbadb45e2a0)

**Comment by sebhildebrandt** [comment]
@DB-47 can you provide the output of the following (from powerShell)

```
Get-CimInstance Win32_PhysicalMemory | select DataWidth,TotalWidth,Capacity,BankLabel,MemoryType,SMBIOSMemoryType,ConfiguredClockSpeed,FormFactor,Manufacturer,PartNumber,SerialNumber,ConfiguredVoltage,MinVoltage,MaxVoltage | fl
```

**Comment by DB-47** [comment]
Here they are: 

![image](https://github.com/sebhildebrandt/systeminformation/assets/34443035/6c429f14-b8e2-4f0e-a4a6-b370ebc28afa)

**Comment by DB-47** [comment]
Should you need more information / context, feel free to ping me again

**Comment by sebhildebrandt** [comment]
@DB-47 ... thank you! This is more than enough ;-) ... I already have an idea how to proceed. Unfortunately Microsofts documentation seems to be outdated. No hint to DDR5 ... but I already found, what I was searching for. Will provide a fix on Monday.

**Comment by sebhildebrandt** [comment]
@DB-47 internal tests already shows correct men type (DDR5) ... new release will be published Monday.

**Comment by sebhildebrandt** [comment]
@DB-47 ... I just published version 5.21.0 which should include the fix. Can you test it on your side?
